golang/stdin.go

```diff
@@ -0,0 +1,23 @@
+// Test script to get input, transform, and write to stdout
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+func main() {
+	scanner := bufio.NewScanner(os.Stdin)
+	counter := 1
+
+	for scanner.Scan() {
+		line := scanner.Text()
+		if line == "" {
+			break
+		}
+		fmt.Printf("%d %s\n", counter, strings.ToUpper(line))
+		counter++
+	}
```

Comment:
```suggestion
	for lineNumber := 1; scanner.Scan(); lineNumber++ {
		fmt.Printf("%d %s\n", lineNumber, strings.ToUpper(scanner.Text()))
	}
```

a more succinct way to do this